src/app/tests/TestClusterInfo.cpp

```diff
@@ -99,11 +99,11 @@ void TestAttributePathIncludedDifferentClusterId(nlTestSuite * apSuite, void * a
 }
 
 /*
-{ClusterInfo::kInvalidEndpointId, ClusterInfo::kInvalidClusterId, ClusterInfo::kInvalidEventId},
-{ClusterInfo::kInvalidEndpointId, MockClusterId(1), ClusterInfo::kInvalidEventId},
-{ClusterInfo::kInvalidEndpointId, MockClusterId(1), MockEventId(1)},
-{kMockEndpoint1, ClusterInfo::kInvalidClusterId, ClusterInfo::kInvalidEventId},
-{kMockEndpoint1, MockClusterId(1), ClusterInfo::kInvalidEventId},
+{kInvalidEndpointId, ClusterInfo::kInvalidClusterId, ClusterInfo::kInvalidEventId},
+{kInvalidEndpointId, MockClusterId(1), ClusterInfo::kInvalidEventId},
+{kInvalidEndpointId, MockClusterId(1), MockEventId(1)},
+{kMockEndpoint1, kInvalidClusterId, ClusterInfo::kInvalidEventId},
+{kMockEndpoint1, MockClusterId(1), kInvalidEventId},
```

Comment:
Why do some `ClusterInfo::` has been removed but not all ? Search and replace of the first occurrence or am I missing something ?

Reply:
Yes, that is exactly what happened.  Fixed.